tests/x11regressions/libreoffice/libreoffice_mainmenu_favorites.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
```

Comment:
as you didn't work on this during 2012-2015, please limit the years of the copyright